sql-plugin/src/main/java/com/nvidia/spark/rapids/catalyst/CudfUnsafeRow.java

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.catalyst;
+package com.nvidia.spark.rapids.catalyst;
 
 import com.nvidia.spark.rapids.GpuColumnVector;
+import org.apache.spark.sql.catalyst.InternalRow;
```

Comment:
There's supposed to be a whitespace between the Spark imports.  See https://github.com/NVIDIA/spark-rapids/blob/branch-21.10/CONTRIBUTING.md#code-formatting and the sample code formatting file.

Reply:
Thanks!  This helped me realize I was not using the correct profile in IntelliJ.